### PR TITLE
Fix the caption to "Relation types to propagate

### PR DESCRIPTION
**Pull request by jktu2870**
## Before
image

## After
Знімок екрана 2024-02-14 125701

Task
PROD-3051

### CODE REVIEW

File: ui-ngx/src/app/modules/home/components/profile/alarm/device-profile-alarm.component.html [hunk 1]
```diff
@@ -73,7 +73,6 @@
                 <mat-icon matChipRemove>close</mat-icon>
               </mat-chip-row>
               <input matInput type="text" placeholder="{{'device-profile.alarm-rule-relation-types-list' | translate}}"
-                     style="max-width: 200px;"
```
Comment (vvlladd28) [inline ui-ngx/src/app/modules/home/components/profile/alarm/device-profile-alarm.component.html:76]: Why do you need to correct styles if translations are corrected in PR?

File: ui-ngx/src/assets/locale/locale.constant-ko_KR.json [hunk 2]
```diff
@@ -1008,7 +1008,7 @@
         "advanced-settings": "Advanced settings",
         "propagate-alarm": "Propagate alarm",
         "alarm-rule-relation-types-list": "Relation types to propagate",
-        "alarm-rule-relation-types-list-hint": "If Propagate relation types are not selected, alarms will be propagated without filtering by relation type.",
+        "alarm-rule-relation-types-list-hint": "If propagate relation types are not entered, alarms will be propagated without filtering by relation type.",
```
Comment (vvlladd28) [inline ui-ngx/src/assets/locale/locale.constant-ko_KR.json:1011]: Maybe it is better to delete the block with English translations?

File: ui-ngx/src/app/modules/home/components/profile/alarm/device-profile-alarm.component.html (hunk 1 above)
Comment (jktu2870) [reply ui-ngx/src/app/modules/home/components/profile/alarm/device-profile-alarm.component.html:76]: max-width cuts off the last letter (same as any letter that makes the placeholder longer than 200px)

**Review by vvlladd28 — APPROVED** [review]

File: ui-ngx/src/assets/locale/locale.constant-sl_SI.json [hunk 3]
```diff
@@ -1008,7 +1008,7 @@
     "advanced-settings": "Advanced settings",
     "propagate-alarm": "Propagate alarm",
     "alarm-rule-relation-types-list": "Relation types to propagate",
-    "alarm-rule-relation-types-list-hint": "If Propagate relation types are not selected, alarms will be propagated without filtering by relation type.",
+    "alarm-rule-relation-types-list-hint": "If propagate relation types are not entered, alarms will be propagated without filtering by relation type.",
```
[the remark this answers is not in the document]
Comment (ashvayka) [reply ui-ngx/src/assets/locale/locale.constant-sl_SI.json:1011]: Defines relation types to filter the related entities. If not set, the alarm will be propagated to all related entities.

File: ui-ngx/src/assets/locale/locale.constant-sl_SI.json [hunk 4]
```diff
@@ -1008,7 +1008,7 @@
     "advanced-settings": "Advanced settings",
     "propagate-alarm": "Propagate alarm",
     "alarm-rule-relation-types-list": "Relation types to propagate",
```
Comment (ashvayka) [inline ui-ngx/src/assets/locale/locale.constant-sl_SI.json:1010]: "Relation types"

**Pull request merged**